src/browser/bounds_changed_state_tracker.js

```diff
@@ -440,6 +442,9 @@ function BoundsChangedStateTracker(uuid, name, browserWindow) {
     var _listeners = {
         'begin-user-bounds-change': () => {
             setUserBoundsChangeActive(true);
+            const cachedBounds = getCachedBounds();
```

Comment:
Do they need / did they ask for the cached bounds somewhere?

Reply:
It was considered a "nice to have".

Reply:
Yes, I believe the idea is to have the starting point before any user-generated moves.  This would probably be a good place to put the modifier keys (shift) as well but that will require native work.

Reply:
will this fire for every window in the group?